test/unit/rules/lint-no-trailing-dot-in-path-expression-test.js

```diff
@@ -1,86 +0,0 @@
-'use strict';
-
-const generateRuleTests = require('../../helpers/rule-test-harness');
-
-generateRuleTests({
-  name: 'no-trailing-dot-in-path-expression',
-  skipDisabledTests: true,
-  config: true,
-
-  good: [
-    '{{contact}}',
-    "<span class={{if contact.is_new 'bg-success'}}>{{contact.contact_name}}</span>",
-    '{{#if contact.contact_name}}\n' + '   {{displayName}}\n' + '{{/if}}',
-    '{{#contact-details contact=contact}}\n' +
-      '   {{contact.displayName}}\n' +
-      '{{/contact-details}}',
-  ],
-
-  bad: [
-    {
-      template: '{{contact.}}',
-
-      fatal: {
```

Comment:
These tests are removed because we now actually have the ability to receive **multiple** fatal errors (we were previously just swallowing them). I removed the tests since this is all just testing implementation details of `@glimmer/syntax` itself.